python/nvtx/_lib/lib.pyx

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from nvtx._lib.lib cimport *
+from nvtx.colors import color_to_hex
+from nvtx.utils.cached import CachedInstanceMeta
+
+cdef extern from "Python.h":
+    wchar_t* PyUnicode_AsWideCharString(object, Py_ssize_t *)
+
+
+def initialize():
+    nvtxInitialize(NULL)
+
+
+cdef class EventAttributes:
```

Comment:
Should we move these class definitions into a pxd so they can be cimported by someone?

Reply:
Done